backend/routers/memories.py

```diff
@@ -14,6 +14,21 @@
 
 router = APIRouter()
 
+@router.put("/v1/memories/{memory_id}/mark-created", response_model=Memory, tags=['memories'])
+def mark_memory_as_created(memory_id: str, event_index: int, uid: str = Depends(auth.get_current_user_uid)):
+
+    memory = _get_memory_by_id(uid, memory_id)
```

Comment:
Ideally here, we only modify memory.structured.events, and rewrite it, instead of the whole thing